alertproduction.tex

```diff
@@ -2,43 +2,58 @@
+Alerts generated during the early science period may be produced with higher latency. 
+Access to images (\S~\ref{sec:prompt-images}) and the PPDB (\S~\ref{sec:ppdb}) will not initially be available.
 
-Rubin aims to scale up alert production during commissioning with the aim of beginning early Alert Production over a progressively increasing fraction of the sky as soon as is feasible following Rubin First Light  (\S~\ref{sec:timeline}).
-\citeds{RTN-061} describes the criteria for sending the first Rubin alerts.
-Once begun, Alert Production will then proceed continuously into the full LSST survey.
-Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB may not be available during this phase.
 During  commissioning and early Operations periods,  alert packets for moving objects might not include the associated historical source records , and parameters such as phase curve slope (G) would be empty until sufficient detections exist to derive them.
-% Table~\ref{tab:prompt-data-products} lists the various alert and prompt processing data products currently planned at each phase of alert production during commissioning and the first two years of the LSST survey.
-%Phase 1 covers the commissioning period and phase 2 covers early survey operations.
-During routine LSST operations, prompt image data products, including  raw images, processed single visit images (PVIs), difference images, and template images, will be made available no earlier than 80 hours following camera readout.
-During the first 6 months of the LSST,  prompt PVIs and difference images may be released with higher latency as Rubin continues to understand data quality and scale up services.
+Solar System discoveries will be reported to the Minor Planet Center on an ad-hoc basis during Early Operations, with a goal of increased automation as alert production becomes routine.
+
+%%% Prompt images
+\subsection{Prompt Images and Catalogs}
+\label{sec:prompt-images}
+
+Starting in early operations and continuing through Data Release 1 (DR1), Rubin plans to begin nightly releases of single-epoch Processed Visit Images (PVIs) together with the associated Source catalogs of detections. 
+These Source catalogs will be provided through the Butler on a per-image basis.
+The PVIs will be subject to the standard 80-hour embargo period, while the catalogs are not but, for convenience and consistency, will be released together with the images after the 80-hour embargo.
+During the first six months of LSST operations, prompt PVIs and difference images may be released with somewhat higher latency as Rubin continues to assess data quality and scale up data services.
+
+Prompt image publication is under active development.  The current expected start date for nightly PVI and catalog releases is not before April 2026. 
```

Comment:
I think you want to take "and catalog" out of here, since PPDB has its own immediately following subsection.

Reply:
I an referring to the Source catalogs of detections in PVIs, not the PPDB catalogs. The Source catalogs will be provided through the Butler on a per-image basis -- not PPDB